backend/src/main/java/com/example/backend/study/api/controller/info/StudyInfoController.java

```diff
@@ -0,0 +1,60 @@
+package com.example.backend.study.api.controller.info;
+
+import com.example.backend.auth.api.service.auth.AuthService;
+import com.example.backend.study.api.service.info.StudyInfoService;
+import com.example.backend.common.response.JsonResult;
+import com.example.backend.domain.define.account.user.User;
+import com.example.backend.domain.define.study.info.StudyInfo;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/auth/studyinfo")
+public class StudyInfoController {
+
+    private final StudyInfoService studyInfoService;
+    private final AuthService authService;
+
+    // 스터디 등록
+    @PostMapping("/register")
```

Comment:
```suggestion
    @PostMapping("")
```

POST 자체가 등록한다는 의미가 있는데, URL 네이밍은 다음과 같이하는건 어떨까요?